### DISCUSSION TITLE
Expand the value proposition, to be even more helpful for designers without any OSS knowledge

### DISCUSSION THREAD

**Discussion by ifeneris** [contribute-design/contribute.design#58] [Ideas]
The current setup and messaging on https://contribute.design/ are great for communicating this simple yet efficient solution (the .design folder and the guidelines) to connect designers with OSS projects and OSS project owners with designers.

However, @fivenp, I was curious if expanding the project’s scope to include onboarding for designers new to OSS would align with its goals. For example:

- Introductory content to highlight the value designers can bring to OSS so that the visitors would be more aware and engaged in searching for projects and issues to contribute
- Onboarding resources like “How-to” guides, FAQs, or learning cards on OSS project processes so that designers without proper knowledge would quickly understand what to do next
- Changing the priority of messages so that we first focus on designers and then on developers/project owners (currently, the options for developers are first, and their CTAs are main, vs those for designers)

If this fits, I’d be happy to open relevant issues to explore design proposals for such solutions.

**Comment by fivenp** [comment]
I really like your thought process here. 

From my perspective (and the learnings) it was more crucial in the beginning to gather OSS projects for this initiative then to find designers willing to participate. 

While I think this is still very valid (since we have close to NO standing and visibility in OSS world – and only a handful of projects actively looking for support) I would LOVE to expand the "For designers" section with all the things you mentioned.  

Maybe we simply need to also rethink the two landing pages based on the target audience and promote them more instead of the main site? :)

**Reply by ifeneris** [reply]
> rethink the two landing pages based on the target audience and promote them more

@fivenp you mean the /for-developers and /for-designers, right? Yeah, totally agree, whatever the expansions, these pages should serve as landing pages, and we should promote them in priority on each specific target audience.

Great, I am opening an issue on that opportunity.
